<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alumno</class>
 <widget class="QDialog" name="alumno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SRC.qrc">
    <normaloff>:/Images/1490820813-6_82408.png</normaloff>:/Images/1490820813-6_82408.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="SRC.qrc">:/Images/skill_102344.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Foto">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="SRC.qrc">:/Images/foldertealphoto_92861.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Cambiar_Foto">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">       background:transparent;
       border:none;
       outline:none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Ver_Perfil">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Berlin Sans FB Demi&quot;;</string>
    </property>
    <property name="text">
     <string>Ver Perfil</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>451</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Alumno">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 25pt &quot;Berlin Sans FB Demi&quot;;
background:transparent;

</string>
    </property>
    <property name="text">
     <string>Alumno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 25pt &quot;Berlin Sans FB Demi&quot;;
background:transparent;

</string>
    </property>
    <property name="text">
     <string>Bienvenido </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="SRC.qrc">:/Images/pencils_120694.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="SRC.qrc">:/Images/laptop-1_102326.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>391</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
  border: 2px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="Visualizar_Avance">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Berlin Sans FB Demi&quot;;</string>
     </property>
     <property name="text">
      <string>Visualizar Avance</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Marcar_Materias">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Berlin Sans FB Demi&quot;;</string>
     </property>
     <property name="text">
      <string>Marcar Mat.Por Cursar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Cambiar_Contrasena">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Berlin Sans FB Demi&quot;;</string>
     </property>
     <property name="text">
      <string>Cambiar contraseña</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Cerrar_Sesion">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>400</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FF4081;
font: 14pt 75 &quot;Times New Roman&quot;;
border:none;
hover!pressed:border: 5px solid blue;
</string>
     </property>
     <property name="text">
      <string>Cerrar sesión</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Avance">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="SRC.qrc">:/Images/checklist_25365.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Marcar">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="SRC.qrc">:/Images/checklist_102320.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Contrasena">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="SRC.qrc">:/Images/door_internet_key_lock_password_safe_security_icon_127075.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Cambio_al">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       background:transparent;
       border:none;
       outline:none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cambio_al_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       background:transparent;
       border:none;
       outline:none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cambio_al_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       background:transparent;
       border:none;
       outline:none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Marcar_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="SRC.qrc">:/Images/checklist_102320.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Marcar_Materias_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Berlin Sans FB Demi&quot;;</string>
     </property>
     <property name="text">
      <string>Marcar Mat.Aprobadas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Marcar_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="SRC.qrc">:/Images/checklist_102320.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Marcar_Materias_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Berlin Sans FB Demi&quot;;</string>
     </property>
     <property name="text">
      <string>Marcar Mat.Cursando</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Cambio_al_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       background:transparent;
       border:none;
       outline:none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cambio_al_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       background:transparent;
       border:none;
       outline:none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Avance_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="SRC.qrc">:/Images/checklist_25365.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Visualizar_Avance_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>350</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Berlin Sans FB Demi&quot;;</string>
     </property>
     <property name="text">
      <string>Proyección de materias</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Cambio_al_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>270</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       background:transparent;
       border:none;
       outline:none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1DE9B6;  
border: 1px solid #1DE9B6;
    border-radius: 20px;
    margin-top: 0.5em;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>Alumno</zorder>
   <zorder>label_5</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>groupBox_4</zorder>
  </widget>
 </widget>
 <resources>
  <include location="SRC.qrc"/>
 </resources>
 <connections/>
</ui>
